azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/ImageVersionServlet.java

```diff
@@ -132,4 +157,38 @@ protected void handlePost(HttpServletRequest req, HttpServletResponse resp, Sess
           "Exception while creating image version metadata. " + e.getMessage());
     }
   }
+
+  private void handleUpdateImageVersion(final HttpServletRequest req,
+      final HttpServletResponse resp, final Session session,
+      final Map<String, String> templateVariableToValue) throws ServletException,
+      IOException {
+    try {
+      String idString = templateVariableToValue.get(IMAGE_VERSION_ID_KEY);
+      log.info("idString: " + idString);
```

Comment:
Nit: Proper log message. Same comment for each log line in this PR wherever needed.

Reply:
Removed this log.